README.md

```diff
@@ -1,28 +1,45 @@
 # JMeter Service
 
+**Before you start**
+
+Please consider using [job-executor-service](https://github.com/keptn-contrib/job-executor-service) and get full control
+of your (load)test commands, allowing you to run different types of (load)testing tools, e.g.
+* [Jmeter](https://artifacthub.io/packages/keptn/keptn-integrations/jmeter)
+* [locust](https://artifacthub.io/packages/keptn/keptn-integrations/locust-service)
+* [k6](https://artifacthub.io/packages/keptn/keptn-integrations/k6-keptn-integration)
+
+---
+
 The *jmeter-service* has been updated with the implementation from the Keptn-Contrib project [jmeter-extended-service](https://github.com/keptn-contrib/jmeter-extended-service). This service provides extended capabilities around custom workload definitions and executions.
 
 ## Compatibility Matrix
 
 |     Keptn Version     | [Jmeter-service Docker Image](https://github.com/keptn-contrib/jmeter-service/pkgs/container/jmeter-service) |
 |:---------------------:|:------------------------------------------------------------------------------------------------------------:|
-|    0.17.0 - 0.18.x    |                               keptn-contrib/jmeter-service:0.18.0                                            |
+| 0.17.0 and older |                              Please use https://github.com/keptn/keptn/releases                              |
+|      0.18.0      |                                     keptn-contrib/jmeter-service:0.18.1                                      |
```

Comment:
Maybe use `0.18.x` since the version number implies that it would be compatible with `0.18.1`.